src/lib/queries/getMonthlyTotals.ts

```diff
@@ -39,9 +39,7 @@ export default async function getMonthlyTotals(
 ): Promise<MonthlyTotals> {
   const startDate = await getEarliestDate();
   interval = interval
-    // For end we set `endOf('month')` because we are doing monthly aggregation
-    // so we want to take into account the whole month
```

Comment:
lie! this makes quantities to not match with absolute totals. It's ok to have partial month data because users may select to see up until 15th of a month